docs/docusaurus.config.js

```diff
@@ -64,6 +64,14 @@ const config = {
       },
       footer: {
         style: 'dark',
+        logo: {
+          alt: 'riok Logo',
+          src: '/img/riok-logo-grayscale.svg',
+          href: 'https://riok.ch',
+          width: '200px',
+        },
+        copyright:
+          'Mapperly is an opensource project of <a href="https://riok.ch">riok</a>.',
```

Comment:
Should be spelled as "open source project"

Reply:
Fixed.